### ISSUE TITLE
[docs] Define, fire, & listen for custom events

### ISSUE THREAD

**Issue by PotterDai** [Polymer/lit-element#473]
Searched through documentation but didn't find any clue.
It should be pretty obvious I guess?
How can I define an event in my custom element, so that I can use it like
`<my-custom-element @my-event=${this.myEventHandler} />` ?
Thank you very much!

**Comment by Christian24** [comment]
`CustomEvent` does what you need. You can find more info here: https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent
But some pseudo code in your custom element:
``` typescript
fireEvent() {
this.dispatchEvent(newCustomEvent('my-event');
}
```

**Comment by PotterDai** [comment]
@Christian24 It works! Thank you very much! =D

**Comment by katejeffreys** [comment]
To be fixed in #413

**Comment by arthurevans** [comment]
Fixed in #413.
